src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs

```diff
@@ -41,7 +41,6 @@ internal class HttpInListener : ListenerHandler
     internal const string ActivityOperationName = "Microsoft.AspNetCore.Hosting.HttpRequestIn";
     internal const string OnStartEvent = "Microsoft.AspNetCore.Hosting.HttpRequestIn.Start";
     internal const string OnStopEvent = "Microsoft.AspNetCore.Hosting.HttpRequestIn.Stop";
-    internal const string OnMvcBeforeActionEvent = "Microsoft.AspNetCore.Mvc.BeforeAction";
```

Comment:
need to remove from [here](https://github.com/open-telemetry/opentelemetry-dotnet/blob/ecb5dc8732e8d42d7f3eb4c2b9aa77d4b55a75b8/src/OpenTelemetry.Instrumentation.AspNetCore/AspNetCoreInstrumentation.cs#L30) as well. That should